build.sbt

```diff
@@ -4,6 +4,57 @@ name := "tsec"
 
 scalaVersion := "2.12.4"
 
+lazy val contributors = Seq(
+  "jmcardon"             -> "Jose Cardona",
+  "rsoeldner"            -> "Robert Soeldner",
+  "hrhino"               -> "Harrison Houghton",
+  "aeons"                -> "Bjørn Madsen",
+  "ChristopherDavenport" -> "Christopher Davenport"
+)
+
+lazy val releaseSettings = Seq(
+  releaseCrossBuild := true,
+  publishTo := {
+    val nexus = "https://oss.sonatype.org/"
+    if (isSnapshot.value)
+      Some("snapshots" at nexus + "content/repositories/snapshots")
+    else
+      Some("releases" at nexus + "service/local/staging/deploy/maven2")
+  },
+  publishArtifact in Test := false,
+  releasePublishArtifactsAction := PgpKeys.publishSigned.value,
+  scmInfo := Some(
+    ScmInfo(
+      url("https://github.com/jmcardon/tsec"),
+      "git@github.com:jmcardon/tsec.git"
+    )
+  ),
+  homepage := Some(url("https://github.com/jmcardon/tsec")),
+  licenses += ("MIT", url("http://opensource.org/licenses/MIT")),
+  publishMavenStyle := true,
+  pomIncludeRepository := { _ =>
+    false
+  },
+  pomExtra := {
+    <developers>
+      {for ((username, name) <- contributors) yield
+      <developer>
+        <id>{username}</id>
+        <name>{name}</name>
+        <url>http://github.com/{username}</url>
+      </developer>
+      }
+    </developers>
+  },
+  useGpg := false,
+  credentials += Credentials(
```

Comment:
Most people put these in local files. With this we can't help you release. As the repository hard-codes the release performer to you.